### ISSUE TITLE
OBS crashing when interacting with VLC Video Source

### ISSUE THREAD

**Issue by chimattaa** [obsproject/obs-studio#5387]
### Operating System Info

Windows 10

### OBS Studio Version

Other

### OBS Studio Version (Other)

27.1.3

### OBS Studio Log URL

https://obsproject.com/logs/MK5k_kXBLkDlqEXL

### OBS Studio Crash Log URL

https://obsproject.com/logs/MK5k_kXBLkDlqEXL

### Expected Behavior

OBS should continue to operate normally when I interact with the VLC Video Source (open settings, stop/play file, move to a new scene with an auto-playing VLC Video source). 

### Current Behavior

OBS immediately crashes. It also crashes when the VLC Video source reaches the end of the media file and should loop back.

### Steps to Reproduce

1. Have a scene with a VLC Video source (I have an mp3) that automatically starts playing and will loop.
2. Open OBS. The mp3 starts playing.
3. Click the VLC video source and press the stop button.
...

### Anything else we should know?

I have logs that show both fault address being either "libvlccore.dll" and sometimes "ntdll.dll"

**Comment by RytoEX** [comment]
What version of VLC do you have installed?

Also, you've uploaded the crash log twice.  The "OBS Studio Log URL" should be the regular OBS log for the session where the crash occurred.

**Comment by chimattaa** [comment]
> What version of VLC do you have installed?
I have 3.0.15.

> Also, you've uploaded the crash log twice. The "OBS Studio Log URL" should be the regular OBS log for the session where the crash occurred.
Sorry, I was confused about the difference between "OBS Studio Log URL" and "OBS Studio Crash Log URL". I see there are two different options now.
Here's my OBS Studio Log URL from the last opened time: https://obsproject.com/logs/yxGsKMvnozHuVHKh

**Comment by chimattaa** [comment]
did a bit more testing.
Let's call the VLC Video source "bgm", as I put mp3s in there.

One thing I can do is right click -> Delete to just remove it out of my source list. When I go to create a new source called "bgm." it allows me to do so without (any apparent) issues, but when I pressed the stop button, OBS crashed so suddenly it didn't even give me a popup about the crash. When I reopened OBS, that newly added "bgm" source is still there. As before, if I interact with it, I crash.
